run_terminate_cloudfront_iam_invaildations.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+from env import env
+from run_common import AWSCli
+from run_common import print_message
+from run_common import print_session
+
+args = []
+
+# It is a script dedicated to the op environment.
+if __name__ == "__main__":
+    from run_common import parse_args
+
+    args = parse_args()
+
+
+def run_terminate_cloudfront_iam_for_invaildations(name):
+    print_session('terminate iam')
+
+    aws_cli = AWSCli()
+
+    iam_owner_aws_account_id = env['cloudfront']['IAM_OWNER_AWS_ACCOUNT_ID']
```

Comment:
@GoldieKMG //  현재 스크립트를 돌리고 있는 대상 계정의 account id는 API로 가져올수 있습니다. 굳이 magi애서 입력받을 필요가 없습니다. https://github.com/HardBoiledSmith/johanna/blob/8521c64df8d172d4eb640cb95e41f619df465431/run_common.py#L230-L233
```suggestion
    iam_owner_aws_account_id = aws_cli.get_caller_account_id()
```